frontend-project/src/pages/events/EventsListView.tsx

```diff
@@ -0,0 +1,113 @@
+import { ActionType, ProColumns } from '@ant-design/pro-table';
+import { Button, Space, Tooltip } from 'antd';
+import React, { useRef } from 'react';
+import { formatMessage } from 'umi-plugin-react/locale';
+
+import { Event } from '@/services/definitions';
+import { EventsService } from '@/services/events';
+import Table from '@/components/Table';
+import router from 'umi/router';
+import { DeleteOutlined, EditOutlined } from '@ant-design/icons';
+import { useDispatch } from 'dva';
+import { Link } from 'umi';
+import { PaginationParams, PaginationResponse } from '@/services/common';
+import { openNotificationWithIcon } from '@/models/global';
+import { ServiceResponse } from '@/services/service';
+import { localeKeys } from '@/locales/pl-PL';
+import CaseName from '@/components/Table/CaseName';
+
+function EventsListView() {
+  const dispatch = useDispatch();
+  const tableActionRef = useRef<ActionType>();
+
+  function onEdit(id: number) {
+    router.push(`/events/edit/${id}`);
+  }
+
+  function onRemove(id: number) {
```

Comment:
Analogiczna uwaga jak do `onRequestDone` i `onFinish`: wyekstrahować kawał wspólny dla wszystkich ListView, i tu zostawić jakiś
```
const onRemove = onRemoveFactory('cases/remove', 'cases');
```

Reply:
Nie chcę przenosić logiki pokazywania błędu do fabryk czy wrapperów na funkcję. Równie dobrze można to przenieść do serwisów reduxowych, ale nie wiem czy chcemy w to iść.